python/rascaline/rascaline/utils/__init__.py

```diff
@@ -3,9 +3,10 @@
 import metatensor
 
 from .power_spectrum import PowerSpectrum
+from .splines import SplinerBase, SplinerFromFunction
 
 
-__all__ = ["PowerSpectrum"]
+__all__ = ["PowerSpectrum", "SplinerBase", "SplinerFromFunction"]
```

Comment:
```suggestion
```

I don't think `__all__` is really useful here.

Reply:
yeah it is not top level...